cmake/auxiliary.sh.in

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+trap '' HUP
+export LD_LIBRARY_PATH="$(dirname "$0")/python_lib/lib-dynload"
+"$(dirname "$0")/energyplus" auxiliary @ARGUMENT@ &
+trap - HUP
```

Comment:
This is a sweet little bash script that allows execution of a script without popping open the terminal.  The shenanigans are top notch but work flawlessly.